bentoml/_internal/bento/docker/Dockerfile.template

```diff
@@ -0,0 +1,32 @@
+FROM {base_image}
+
+ARG UID=1034
+ARG GID=1034
+RUN groupadd -g $GID -o bentoml && useradd -m -u $UID -g $GID -o -r bentoml
+
+ARG BENTO_PATH=/home/bentoml/bento
+ENV BENTO_PATH=$BENTO_PATH
+ENV BENTOML_HOME=/home/bentoml/
+
+RUN mkdir $BENTO_PATH && chown bentoml:bentoml $BENTO_PATH -R
+WORKDIR $BENTO_PATH
+
+# copy over env directory
+COPY --chown=bentoml:bentoml ./env ./env
+RUN chmod +x ./env/docker/init.sh
+RUN ./env/docker/init.sh ensure_python
+RUN ./env/docker/init.sh restore_conda_env
+RUN ./env/docker/init.sh install_pip_packages
+RUN ./env/docker/init.sh install_wheels
+RUN ./env/docker/init.sh user_setup_script
+
+# copy over all remaining bento files
+COPY --chown=bentoml:bentoml . ./
+
+# Default port for BentoML Service
+EXPOSE 5000
+
+USER bentoml
+RUN chmod +x ./env/docker/entrypoint.sh
+ENTRYPOINT [ "./env/docker/entrypoint.sh" ]
+CMD ["bentoml", "serve", "."]  # TODO: add --production flag
```

Comment:
cc @bojiang on the TODO

Reply:
we can have it right now

Reply:
even the current serve-production subcommand works

Reply:
it doesn’t work due to the cwd in watchers change in the previous PR

Reply:
```suggestion
CMD ["bentoml", "serve", ".", "--production"]
```